frontend/src/assets/css/typography.scss

```diff
@@ -1,3 +1,5 @@
+@import "./vars.scss";
```

Comment:
why do we need that ?

Reply:
I had to add this because if I don't specify it, the app doesn't work.
src\assets\css\typography.scss   root stylesheet
  Error: Undefined variable.